### PR TITLE
product card was added

### PR DESCRIPTION
**Pull request by Maksym845**
[DEMO LINK](https://Maksym845.github.io/layout_product-cards/)
[TEST REPORT LINK](https://Maksym845.github.io/layout_product-cards/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -1,13 +1,50 @@
...
+  </head>
+  <body class="page__body">
+    <div class="card" data-qa="card">
+      <div class="card__image-wrap">
+        <img
+          src="images/imac.jpeg"
+          alt="imac image"
+          class="card__image"
+        >
+      </div>
```
Comment (anastasiia-nahaiska) [inline src/index.html:22]: I think that this wrap is unnecessary

File: src/index.html [hunk 2]
```diff
@@ -1,13 +1,50 @@
...
+      <div class="card__content">
```
Comment (anastasiia-nahaiska) [inline src/index.html:23]: [CODE STYLE] - Add empty lines between multiline sibling blocks of HTML. But don't add empty lines between parent and child elements.

You also have this problem below

File: src/index.html [hunk 3]
```diff
@@ -1,13 +1,50 @@
...
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <p class="card__reviews">Reviews: 5</p>
+        </div>
+        <div class="card__price">
+          <p class="card__code card__code--left">Price:</p>
+          <p class="card__code card__code--right">$2,199</p>
+        </div>
```
Comment (anastasiia-nahaiska) [inline src/index.html:41]: The guys who reviewed my code gave me a good solution for this case. You can replace div -> p and p ->span. In this case, we don't use block div and only have one p

File: src/card.css [hunk 4]
```diff
@@ -0,0 +1,124 @@
...
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+
+  width: 166px;
+
+  padding-left: 16px;
+  padding-right: 16px;
+  padding-top: 32px;
+  padding-bottom: 16px;
```
Comment (anastasiia-nahaiska) [inline src/card.css:19]: You can only have one line 
padding: 32px 16px 16px;

File: src/card.css [hunk 5]
```diff
@@ -0,0 +1,124 @@
...
+  margin: 0;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+
+  width: 166px;
+
```
Comment (anastasiia-nahaiska) [inline src/card.css:15]: You can use   box-sizing: border-box; and then use width: 200px. In this case, the width will be calculated taking into account the internal indents

File: src/card.css [hunk 6]
```diff
@@ -0,0 +1,124 @@
...
+  border-radius: 5px;
+}
+
+.card__image-wrap {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+
+  padding: 0 3px;
+  padding-bottom: 40px;
```
Comment (anastasiia-nahaiska) [inline src/card.css:31]: padding: 0 3px 40px;

File: src/card.css [hunk 7]
```diff
@@ -0,0 +1,124 @@
...
+}
+
+.card__image {
+  height: 134px;
+  width: 160px;
+}
+
+.card__title {
+  margin: 0;
```
Comment (anastasiia-nahaiska) [inline src/card.css:40]: You can create a separate reset.css file for all reset actions

File: src/card.css [hunk 8]
```diff
@@ -0,0 +1,124 @@
...
+
+.card__reviews {
+  margin: 0;
+  display: flex;
+  align-items: flex-end;
+
+  font-size: 10px;
+  line-height: 14px;
+  color: #060b35;
+  font-weight: 400;
```
Comment (anastasiia-nahaiska) [inline src/card.css:75]: You can add these styles to .card and replace some styles when needed. In this case, you will have fewer lines of code and repeats

**Review by anastasiia-nahaiska — COMMENTED** [review]
Good job!!! I leave several comments, maybe it will be useful for you

File: src/card.css (hunk 7 above)
Comment (Maksym845) [reply src/card.css:40]: I know, but the task says we must have only two blocks, the card, and stars.

File: src/card.css (hunk 6 above)
Comment (Maksym845) [reply src/card.css:31]: Rly useful, I didn`t know about this solution, thank you!

File: src/index.html [hunk 9]
```diff
@@ -1,13 +1,52 @@
...
+        
+        <div class="card__price">
+          <p class="card__code card__code--left">Price:</p>
+          <p class="card__code card__code--right">$2,199</p>
+        </div>
+        <a
+          href="/"
+          class="card__button_link"
+          data-qa="hover"
+        >Buy</a>
```
Comment (Anna-Tkachenko) [inline src/index.html:48]: Fix indents
```suggestion
        <a
          href="/"
          class="card__button_link"
          data-qa="hover"
        >
          Buy
        </a>
```

File: src/index.html (hunk 3 above)
Comment (Anna-Tkachenko) [reply src/index.html:43]: @anastasiia-nahaiska good point👍

**Review by Anna-Tkachenko — CHANGES_REQUESTED** [review]

File: src/index.html [hunk 10]
```diff
@@ -1,13 +1,50 @@
...
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <div class="card" data-qa="card">
+      <img
+        src="images/imac.jpeg"
+        alt="imac image"
+        class="card__image"
+      >
+      <h1 class="card__title">
```
Comment (Anna-Tkachenko) [inline src/index.html:21]: h1 usually used for the one general website title
for the card title will be better to use h2 or h3

**Review by Anna-Tkachenko — APPROVED** [review]
Good job!

**Pull request closed without merging**